dvc/main.py

```diff
@@ -37,7 +38,10 @@ def main(argv=None):
             logger.setLevel(logging.DEBUG)
 
         cmd = args.func(args)
-        ret = cmd.run_cmd()
+        ret = cmd.run()
+    except LockError:
+        logger.exception("failed to lock before running a command")
```

Comment:
Should we show more descriptive and actionable message? Like what is happening, i.e. that some concurrent dvc command is blocking. And if that is no so then what lock files should people remove.

Reply:
@Suor Sure https://github.com/iterative/dvc/issues/2520